packages/webpack-cli/lib/utils/cli-flags.js

```diff
@@ -33,7 +33,7 @@ let flagsFromCore =
         : [];
 
 // duplicate flags
-const duplicateFlags = ['entry', 'mode', 'stats', 'watch', 'target', 'devtool'];
+const duplicateFlags = ['entry', 'mode', 'stats', 'watch', 'target', 'devtool', 'name'];
```

Comment:
Can we use `Object.keys()` on array flags for webpack@4 to avoid it in future?

Reply:
Yeah we should do that to avoid manual work every time

Reply:
Done 💯